Dockerfile

```diff
@@ -187,7 +187,7 @@ ENV CC="riscv64-linux-gnu-gcc -static"
 RUN cmake . && make && mkdir /out/ && mv tini /out/
 
 FROM ubuntu:22.04 AS rootfs-riscv64-dev
-RUN apt-get update -y && apt-get install -y mkisofs
+RUN apt-get update -y && apt-get install -y squashfs-tools
```

Comment:
Squashfs comes with some overhead, right?. Can we default to isofs and enable squashfs only for d2oci usage?

Reply:
Added iso back. Added erofs (faster as it aligns data).